server/models/player.py

```diff
@@ -9,6 +9,7 @@ class Player(db.Model):
     team_id = db.Column(db.Integer, db.ForeignKey('team.id'))
     name = db.Column(db.Text)
     gender = db.Column(db.Text)
+    fallback_salary = db.Column(db.Integer)
```

Comment:
nice! I seem to remember there was a work around way of doing this at one time but this is much better!